src/app/ui/components/button/button.tsx

```diff
@@ -10,11 +10,22 @@ export type ButtonProps = Omit<
   ButtonVariantProps;
 
 export function Button(props: ButtonProps) {
-  const { children, fullWidth, size, trigger, invert, type = 'button', variant, ...rest } = props;
+  const {
+    children,
+    fullWidth,
+    size,
+    trigger,
+    invert,
+    type = 'button',
+    variant,
+    disabled,
+    ...rest
+  } = props;
   return (
     <StyledButton
       className={buttonRecipe({ fullWidth, size, invert, trigger, variant })}
       type={type}
+      css={disabled ? { pointerEvents: 'none' } : {}}
```

Comment:
It would be better to do this in the `button-recipe`

Reply:
Also think we should just change styled such than nothing happens, rather than setting pointerEvents none.

E.g. a disable button might have a tooltip